templates/trainer.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'base.html' %}
+{% block title %}
+Trainers
+{% endblock %}
+
+{% block content %}
+<table class="table table-hover table-dark">
+  <thead>
+    <tr>
+      <!-- Your Columns HERE -->
+      <th scope="col">Username</th>
+      <th scope="col">Password</th>
+      <th scope="col">Firstname</th>
+      <th scope="col">Lastname</th>
+      <th scope="col">DOB</th>
+    </tr>
+  </thead>
+  <tbody>
+    <!-- Your rows inside the table HERE: -->
+    {% for trainer in results %}
+    <tr>
+      <td>{{ trainer[0] }}</td>
+      <td>{{ trainer[1] }}</td>
+      <td>{{ trainer[2] }}</td>
+      <td>{{ trainer[3] }}</td>
+      <td>{{ trainer[4] }}</td>
+      <td>
+      <td>
+      <button class="btn btn-outline btn-info">Something ? </button>
```

Comment:
What is the functionality of this button?

Reply:
Place holder button for future functionality